cayleypy/graphs_lib_test.py

```diff
@@ -21,6 +21,80 @@ def test_pancake():
     ))
 
 
+def test_cubic_pancake():
+
+    graph = prepare_graph("cubic_pancake", n=15, S=1)
```

Comment:
Can use lowercase "s"? I know this is nitpicking, but Python has standard conventions on naming, and variable names must be lowercase.
https://peps.python.org/pep-0008/#function-and-variable-names

P.S. I am surprised why linter didn't cacth this. Note to self: add autopep8 to automatic checks.